apps/etterlatte-vedtaksvurdering/src/main/resources/db/migration/V16__add_vedtaktype_column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE vedtak ADD COLUMN vedtaktype VARCHAR;
```

Comment:
Hvorfor ikke bare kalle det for `type`?

Reply:
kan godt kalle det type siden vedtak ikke gir noen ekstra info som vi ikke allerede vet utifra context her 👍